x-pack/metricbeat/module/aws/aws.go

```diff
@@ -108,13 +108,13 @@ func getRegions(svc ec2iface.ClientAPI) (completeRegionsList []string, err error
 }
 
 // StringInSlice checks if a string is already exists in list
```

Comment:
Add also a comment about what the new returned int means, or name the returned values.